bower.json

```diff
@@ -33,8 +33,16 @@
     "lodash": "~3.10.1",
     "components-font-awesome": "~4.5.0",
     "angular-poller": "~0.4.5",
-    "moment": "~2.11.2",
-    "pusher": "^3.0.0"
+    "moment": "^2.11.2",
+    "angular-toastr": "^1.7.0",
+    "pusher": "^3.0.0",
+    "angular-translate": "^2.11.0",
+    "angular-translate-handler-log": "^2.11.0",
+    "angular-translate-loader-static-files": "^2.11.0",
+    "angular-translate-storage-cookie": "^2.11.0",
```

Comment:
Shouldn't this dependency be passed in `devDependencies` as the host frontend project will be the one setting the `$translate.use()` and managing storing strategy? (eg, in mnoe-angular the locale key to use it in the uri)